app/controllers/users_controller.rb

```diff
@@ -14,7 +16,7 @@ def create
     authorize!(:create, @user)
 
     if @user.save
-      @user.send_reset_password_instructions
+      deliver_reset_password_instructions(@user)
```

Comment:
it would be good to have this method called form the user object

Reply:
I have removed password_helpers and this method is now called from the user object.